library/entropy_poll.c

```diff
@@ -18,15 +18,9 @@
  */
```

Comment:
From what I understood, this is an issue when the library is built as a dynamic library thus it would be good to me for this to appear in the commit message. Could you add that?

Reply:
No, it's an issue when dynamically linking against glibc. Which is the normal case for glibc.

Reply:
Not in the embedded world.

Reply:
It's common to link statically with uclibc, musl, etc. But with glibc? IIRC some of the glibc functionality (the NSS stuff) doesn't even work when linking statically.

Reply:
ok thus glibc is a non-embedded world library.

Reply:
Define embedded... Is a Raspberry Pi embedded? Raspbian ships with glibc and links to it dynamically.

Reply:
ok thus the only thing we can say is that the must common way (if not the only one) to use glibc is to link dynamically to it?